Fix for issue #1517 , Burplesmirk and Throgg skill issues

floskan opened the pull request.

floskan committed:
Fix for issue #1517 , Burplesmirk and Throgg skill issues

mfvreeland commented:
I understand the Throgg skill line fix part of this, but what exactly is the other part doing? The thread on the forums makes it sound like there's an issue with the settlement razing counter.

floskan commented:
I added a separate issue for that. Burplesmirk should, as all lords, only need 4 skill points to reach the next part of his skill tree blue line. In vanilla right now you need to max out all blue lines skill in the "first part" besides one before you can go into lightning strike and move on. I just changed it to work like it does for Throgg or Wulfrik, only needing 4 points.
